layouts/layout-basics/src/exercises/exercise2/solution/positioning-solution.css

```diff
@@ -0,0 +1,121 @@
+/* 
+  👉 Question 1
+
+  🛠️ Add the necessary CSS properties to position the red dot in the center of the blue rectangle
+
+  ❗ Solution
+
+  ✅ Add position: relative to blue rectange
+  ✅ Add position: absolute to red dot
+  ✅ Specify exact left/right values
+
+*/
+.blue-rectangle {
+  background-color: cornflowerblue;
+  height: 100px;
+  width: 200px;
+  position: relative;
+}
+
+.red-dot {
+  background-color: crimson;
+  border-radius: 20px;
+  height: 20px;
+  width: 20px;
+  position: absolute;
+  top: 40px;
+  left: 80px;
+
+  /*
+    ❗ Alternate Solution 1
+
+  ✅ Add position: relative to blue rectange
+  ✅ Add position: absolute to red dot
+  ✅ Set top/left/right/bottom to 0
+  ✅ Add margin: auto
+
+  */
+  /*
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    margin: auto;
+
+  */
+  /*
+    ❗ Alternate Solution 2
+
+  ✅ Add position: relative to blue rectange
```

Comment:
Do you mean to come back to these comments and change them later? I don't see any CSS in the solution for `position`.